.github/workflows/deploy-docker.yml

```diff
@@ -0,0 +1,134 @@
+name: Deploy Docker
+
+on:
+  workflow_dispatch:
+    branches: ["v31"]
+    inputs:
+      tag:
+        description: tag/version to deploy
+        required: true
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: deploy docker
+        run: |
+          SC_RELEASE_TAG="${{ env.TAG }}"
+          echo "$SC_RELEASE_TAG"
+          
+          TOKEN="${{ secrets.RANCHER2_BEARER_TOKEN }}"
+          RANCHER_HOST="rancher.tools.swagger.io"
+          CLUSTER_ID="c-n8zp2"
+          NAMESPACE_NAME="swagger-oss"
+          K8S_OBJECT_TYPE="daemonsets"
+          K8S_OBJECT_NAME="swagger-petstore-31"
+          DEPLOY_IMAGE="swaggerapi/swagger-petstore31:$SC_RELEASE_TAG"
```

Comment:
check tags and image name in dockerhub, this is wrong, should be `swaggerapi/petstore31`. Also wrong in corresponding file in master and possibly v2